### ISSUE TITLE
Use GitHub as source

### ISSUE THREAD

**Issue by kiliankoe** [kiliankoe/apodidae#13]
Instead of querying Package Catalog, just use the GitHub API to search for swiftlang repos that include a Package.swift. Would be cool to explore how well this works out.

**Comment by kiliankoe** [comment]
After some playing around I came up with this GraphQL query for GitHub's API.

```graphql
query ($query: String!) {
  search(query: $query, type: REPOSITORY, first: 100) {
    repositoryCount
    repositories: edges {
      node {
        ... on Repository {
          nameWithOwner
          description
          url
          isFork
          parent {
            nameWithOwner
          }
          isPrivate
          pushedAt
          license
          openIssues: issues(first: 0, states: OPEN) {
            totalCount
          }
          stargazers(first: 0) {
            totalCount
          }
          tags: refs(refPrefix: "refs/tags/", last: 5) {
            edges {
              node {
                name
              }
            }
          }
          object(expression: "master:Package.swift") {
            ... on Blob {
              abbreviatedOid
            }
          }
        }
      }
    }
  }
  rateLimit {
    remaining
    resetAt
  }
}
```

```json
{
  "query": "promisekit language:swift"
}
```

Filtering for the existence of a Package.swift's objectID is going to have to be done client-side. Only downside here is that a branch called master has to exist, but I guess that's ok.

**Comment by kiliankoe** [comment]
closed with #15
